plugins/auth/src/auth-oidc/index.js

```diff
@@ -131,12 +131,7 @@ export default class openIdConnect {
         case 'login_required':
         case 'account_selection_required':
         case 'consent_required': // possible cause: disabled third party cookies in the browser
```

Comment:
// possible cause: disabled third party cookies in the browser
consent_required case seems to have been added due to 'disabled third party cookies in the browser' . Does this mean that we do not need this case, or the adjustment below should be enough ?

Reply:
In my opinion we should leave this case and just adjust the action - to be sure backward compatibility is kept